tests/integration/test_installation.py

```diff
@@ -192,6 +233,27 @@ def test_repair_run_workflow_job(ws, mocker, new_installation, sql_backend):
     assert run_status.value == "SUCCESS"
 
 
+@retried(on=[NotFound, Unknown, InvalidParameterValue], timeout=timedelta(minutes=8))
+def test_running_real_assessment_job_new_cluster_with_policy(
```

Comment:
These are very long to run. Why new test?

Reply:
removed duplicate and added one with new cluster without override, tested it manually and then marked it as skipped